core/src/main/java/org/apache/accumulo/core/fate/Fate.java

```diff
@@ -403,55 +401,14 @@ public FateId startTransaction() {
 
   public Optional<FateId> seedTransaction(String txName, FateKey fateKey, Repo<T> repo,
       boolean autoCleanUp, String goalMessage) {
-
-    Optional<FateTxStore<T>> optTxStore = store.createAndReserve(fateKey);
-
-    return optTxStore.map(txStore -> {
-      var fateId = txStore.getID();
-      try {
-        Preconditions.checkState(txStore.getStatus() == NEW);
-        seedTransaction(txName, fateId, repo, autoCleanUp, goalMessage, txStore);
-      } finally {
-        txStore.unreserve(Duration.ZERO);
-      }
-      return fateId;
-    });
-  }
-
-  private void seedTransaction(String txName, FateId fateId, Repo<T> repo, boolean autoCleanUp,
-      String goalMessage, FateTxStore<T> txStore) {
-    if (txStore.top() == null) {
-      try {
-        log.info("Seeding {} {}", fateId, goalMessage);
-        txStore.push(repo);
-      } catch (StackOverflowException e) {
-        // this should not happen
-        throw new IllegalStateException(e);
-      }
-    }
-
-    if (autoCleanUp) {
-      txStore.setTransactionInfo(TxInfo.AUTO_CLEAN, autoCleanUp);
-    }
-
-    txStore.setTransactionInfo(TxInfo.TX_NAME, txName);
-
-    txStore.setStatus(SUBMITTED);
+    return store.seedTransaction(txName, fateKey, repo, autoCleanUp);
```

Comment:
fixed in https://github.com/apache/accumulo/commit/e47089c5d9e0efa0007c88b535e9bb3f55b4a44a

Reply:
Just making sure, did you choose to get rid of the goalMessage here since the same message will be logged after this call in both use cases (a system initiated split or a commit compaction). If so, that seems good to me. I was just confused at first why it was removed here then noticed both use cases have a log after.

Reply:
> Just making sure, did you choose to get rid of the goalMessage here since the same message will be logged after this call in both use cases (a system initiated split or a commit compaction). If so, that seems good to me. I was just confused at first why it was removed here then noticed both use cases have a log after.

Yes that is why I removed it.  Then those log messages were removed in 89b70da.  Looking into this a bit more I think the fateKey contains all the information that was in the goal.  The FateLogger class nicely logs the fateKey an other info already, so the information is still available in the logs.  Improved fate logger in 42f8b4d after looking into this comment.

Reply:
I agree that FateKey contains the same info. Those changes look good.